Modelica/SIunits.mo

```diff
@@ -1756,72 +1756,73 @@ argument):</p>
-    Complex(redeclare Modelica.SIunits.ElectricFieldStrength re,
-             redeclare Modelica.SIunits.ElectricFieldStrength im)
+    Complex(redeclare Modelica.SIunits.ElectricFieldStrength re "Real part of complex electric field strength",
+            redeclare Modelica.SIunits.ElectricFieldStrength im "Imaginary part of complex electric field strength")
     "Complex electric field strength";
   operator record ComplexElectricFluxDensity =
-    Complex(redeclare Modelica.SIunits.ElectricFluxDensity re,
-             redeclare Modelica.SIunits.ElectricFluxDensity im)
+    Complex(redeclare Modelica.SIunits.ElectricFluxDensity re "Real part of complex electric flux density",
+            redeclare Modelica.SIunits.ElectricFluxDensity im "Imaginary part of complex electric flux density")
     "Complex electric flux density";
   operator record ComplexElectricFlux =
-    Complex(redeclare Modelica.SIunits.ElectricFlux re,
-             redeclare Modelica.SIunits.ElectricFlux im)
+    Complex(redeclare Modelica.SIunits.ElectricFlux re "Real part of complex electric flux",
+            redeclare Modelica.SIunits.ElectricFlux im "Imaginary part of complex electric flux")
     "Complex electric flux";
   operator record ComplexMagneticFieldStrength =
-    Complex(redeclare Modelica.SIunits.MagneticFieldStrength re,
-             redeclare Modelica.SIunits.MagneticFieldStrength im)
+    Complex(redeclare Modelica.SIunits.MagneticFieldStrength re "Real part of complex magnetic field strength",
+            redeclare Modelica.SIunits.MagneticFieldStrength im "Imaginary part of complex magnetic field strength")
     "Complex magnetic field strength";
   operator record ComplexMagneticPotential =
-    Complex(redeclare Modelica.SIunits.MagneticPotential re,
-             redeclare Modelica.SIunits.MagneticPotential im)
+    Complex(redeclare Modelica.SIunits.MagneticPotential re "Real part of complex magnetic potential",
+            redeclare Modelica.SIunits.MagneticPotential im "Imaginary part of complex magnetic potential")
     "Complex magnetic potential";
   operator record ComplexMagneticPotentialDifference =
-    Complex(redeclare Modelica.SIunits.MagneticPotentialDifference re,
-             redeclare Modelica.SIunits.MagneticPotentialDifference im)
+    Complex(redeclare Modelica.SIunits.MagneticPotentialDifference re "Real part of complex magnetic potential difference",
+            redeclare Modelica.SIunits.MagneticPotentialDifference im "Imaginary part of complex magnetic potential difference")
     "Complex magnetic potential difference";
   operator record ComplexMagnetomotiveForce =
-    Complex(redeclare Modelica.SIunits.MagnetomotiveForce re,
-             redeclare Modelica.SIunits.MagnetomotiveForce im)
+    Complex(redeclare Modelica.SIunits.MagnetomotiveForce re "Real part of complex magnetomotive force",
+            redeclare Modelica.SIunits.MagnetomotiveForce im "Imaginary part of complex magnetomotive force")
```

Comment:
Spelling: magneto motive or magnetomotive?

Reply:
Hmm.... either way I guess, see https://www.collinsdictionary.com/dictionary/english/magnetomotive-force and https://en.wikipedia.org/wiki/Magnetomotive_force. Since we spelled it ```MagnetomotiveForce```, not ```MagnetoMotiveForce```, in this context, I suppose that "magnetomotive force" is OK. I am, however, sure, that we have a couple different spellings in the MSL. So this issue is comparable to the multiphase/multi phase/multi-phase/poly phase discussion. 

For now, I would go for magnetomotive force, and create a ticket, in case you want to initiate a discussion on the unification of the spelling of this term.

Reply:
It is just that it is spelled differently next to each other.
https://github.com/christiankral/Modelica/blob/2a86acdc9d427ed006a452d7a1f9ebfb9612d386/Modelica/SIunits.mo#L1811-L1813

Reply:
I see your point. The term "magnetomotive" is more often used than "magneto motive" in publications of http://ieeexplore.ieee.org. I propose: let's switch to "magnetomotive" on the long run and start change the naming. Would you mind to switch the doc string of the model "magnetomotive" before/after approving the pull request? Thanx.